misty.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.rdoc_options = ['--charset=UTF-8']
   spec.extra_rdoc_files = %w[README.md LICENSE.md]
 
-  spec.add_dependency 'json', '~> 2.0'
+  spec.add_dependency 'json', '>= 1.8.3', '< 3'
```

Comment:
The pessimistic operator is not the same. WIth `~> 1.8`, versions 2.x would not be allowed any more, thus potentially breaking existing users that need `json >= 2.0`, and potentially prohibiting us too from moving to a newer json gem when we update Rails next time.